packages/eas-cli/src/update/__tests__/getBranchNameFromChannelNameAsync-test.ts

```diff
@@ -121,6 +121,7 @@ function mockUpdateBranches(branchNames: string[]): UpdateBranch[] {
           message: 'my message',
           createdAt: '2022-12-07T02:24:43.487Z',
           runtimeVersion: '1.0.0',
+          runtime: {} as Runtime, // Temporary fix to resolve type errors
```

Comment:
This change was introduced in graphql API in https://github.com/expo/universe/commit/3e1eec1811f3256c5e8d5dcb5ba6a3cb6ea80f7c
Not sure how it should be addressed, so for now just casting it 

cc @quinlanj